model/CHEM_DRV.F90

```diff
@@ -438,107 +438,136 @@ SUBROUTINE DO_CHEM
+             State_Met%PFILSAN     (II,JJ,K) = pfilsan(i,j,k)
 
              ! Dwn flux liq prec:conv [kg/m2/s]
-             State_Met%PFLCU       (II,JJ,K) = pflcu(i,j,k)                                             
+             State_Met%PFLCU       (II,JJ,K) = pflcu(i,j,k)
 
              ! Dwn flux ice prec:LS+anv [kg/m2/s]
-             State_Met%PFLLSAN     (II,JJ,K) = pfllsan(i,j,k)                                           
+             State_Met%PFLLSAN     (II,JJ,K) = pfllsan(i,j,k)
 #endif
 
              ! Ice mixing ratio [kg/kg dry air]
-             State_Met%QI          (II,JJ,K) = qci(i,j,k)                                               
+             State_Met%QI          (II,JJ,K) = qci(i,j,k)
 
              ! Water mixing ratio [kg/kg dry air]
-             State_Met%QL          (II,JJ,K) = qcl(i,j,k)                                               
+             State_Met%QL          (II,JJ,K) = qcl(i,j,k)
 
 #ifdef CALC_MERRA2_LIKE_DIAGS
              ! Evap of precip conv [kg/kg/s] (assume per dry air)
-             State_Met%REEVAPCN    (II,JJ,K) = reevapcn(i,j,k)                                          
+             State_Met%REEVAPCN    (II,JJ,K) = reevapcn(i,j,k)
 
              ! Evap of precip LS+anvil [kg/kg/s] (assume per dry air)
-             State_Met%REEVAPLS    (II,JJ,K) = reevapls(i,j,k)                                          
+             State_Met%REEVAPLS    (II,JJ,K) = reevapls(i,j,k)
 #endif
 
              ! Relative humidity [%]
              State_Met%RH          (II,JJ,K) = 100.*q(i,j,k)/QSAT(t(i,j,k)*pk(k,i,j),LHE,pmid(k,i,j))   
              IF ( IT_IS_NAN( State_Met%RH(II,JJ,K) ) ) THEN
                 WRITE(6,*) II,JJ,K, q(i,j,k), QSAT(t(i,j,k)*pk(k,i,j),LHE,pmid(k,i,j)), &
-                     t(i,j,k), pk(k,i,j), LHE, pmid(k,i,j)               
+                     t(i,j,k), pk(k,i,j), LHE, pmid(k,i,j)
                 CALL STOP_MODEL("Bad RH",255)
              ENDIF
 
              ! Specific humidity [g H2O/kg tot air]
-             State_Met%SPHU        (II,JJ,K) = q(i,j,k)                                                 
+             State_Met%SPHU        (II,JJ,K) = q(i,j,k)
 
              ! Specific humidity at start of timestep [g/kg]
-             State_Met%SPHU1       (II,JJ,K) = q(i,j,k)                                                 
+             State_Met%SPHU1       (II,JJ,K) = q(i,j,k)
 
              ! Specific humidity at end of timestep [g/kg]  
-             State_Met%SPHU2       (II,JJ,K) = q(i,j,k)                                                 
+             State_Met%SPHU2       (II,JJ,K) = q(i,j,k)
 
              ! Temperature [K]
-             State_Met%T           (II,JJ,K) = t(i,j,k)*pk(k,i,j)                                       
+             State_Met%T           (II,JJ,K) = t(i,j,k)*pk(k,i,j)
+
+             ! Potential temperature [K]
+             ! TODO: State_Met%THETA       (II,JJ,K) = ???
```

Comment:
This field is still TODO.

Reply:
@jwallwork23, let me do these fields right away. Thanks for the heads up!

Reply:
Apologies, these TODOs were for myself.

Unfortunately, what you pushed in https://github.com/fetch4/GISS-GC/pull/99/commits/cb3cba6cdf0e339aecc6534fc9f4b0971425cd71 won't work because the arrays on the RHS need to come from the GISS model. In each case, we need to:
1. Find out what the array is called in the GISS model.
2. Add a `USE <module>, ONLY: <array>` syntax at the top of the subroutine to import the array.
3. Add the indexed array on the RHS of the expression.

You'll notice that the CI builds failed with your commit because the arrays you added haven't been imported (and might not even have those names anyway).